code/modules/religion/religion_sects.dm

```diff
@@ -213,7 +216,106 @@
 	if(!offering.lit)
 		to_chat(user, "<span class='notice'>The candle needs to be lit to be offered!</span>")
 		return
-	to_chat(user, "<span class='notice'>Another candle for [GLOB.deity]'s collection</span>")
+	to_chat(user, "<span class='notice'>Another candle for [GLOB.deity]'s collection.</span>")
 	adjust_favor(20, user) //it's not a lot but hey there's a pacifist favor option at least
 	qdel(offering)
 	return TRUE
+
+/datum/religion_sect/greed
+	name = "Greedy God"
+	desc = "A very mercantile sect."
+	altar_icon_state = "convertaltar-yellow"
+	alignment = ALIGNMENT_EVIL //greed is not good wtf
+	rites_list = list(/datum/religion_rites/greed/vendatray, /datum/religion_rites/greed/custom_vending)
+	convert_opener = "\"Greed is good.\"<br>\
+	In the eyes of your mercantile deity, your wealth is your favor. Earn enough wealth to purchase some more business opportunities."
+	altar_icon_state = "convertaltar-red"
+
+/datum/religion_sect/honorbound
+	name = "Honorbound God"
+	desc = "A sect that does not engage in harm."
+	altar_icon_state = "convertaltar-white"
+	alignment = ALIGNMENT_GOOD
```

Comment:
I feel like this, given your description, might be more lawful? Just declaring someone as not innocent sounds kinda like some W40k shit haha.

Reply:
I have no idea where cobby is going with alignment so i'm not gonna expand on it too much